be/src/block_cache/disk_space_monitor.cpp

```diff
@@ -168,8 +168,14 @@ void DiskSpaceMonitor::_init_spaces_by_cache_dir() {
     _total_cache_usage = 0;
     for (auto& dir_space : _dir_spaces) {
         auto ret = _fs->directory_size(dir_space.path);
```

Comment:
Have you ever noticed that regular scanning of all files from the file system fills the disk with random IO?

Reply:
> Have you ever noticed that regular scanning of all files from the file system fills the disk with random IO?

There will indeed be some performance overhead here, but I don't think it will have a significant impact. On the one hand, each file is 10GB in size, so the number of files is usually not particularly large. On the other hand, we only scan the block files when the process start, after the system is running, this operation will no longer be performed.